### ISSUE TITLE
Homepage mentions "3kb" but it's 2.5x bigger…

### ISSUE THREAD

**Issue by davidhund** [matthieua/WOW#177]
You might want to update the homepage. 

http://mynameismatthieu.com/WOW/index.html mentions "3kb only" but the current `/dist/wow.min.js` is *8.182 kb* — that's more than 2.5 times as large. Wow. Such bytes :)

People `gzip`, of course, but it's still a little bit misleading because the filesize is advertised as a USP. 
E.g. http://scrollrevealjs.org/ is about the same size.

**Comment by attilaolah** [comment]
Yep, that was written before some new features were added… I'll update the page.
